### PR TITLE
Make copyright year fakable in golden tests

### PR DESCRIPTION
**Pull request by llucax**
We need this to avoid having golden tests fail when the year changes.

Happy new year, everyone!

### CODE REVIEW

**Comment by leandro-lucarella-frequenz** [comment]
I will force-merge because tests are broken without this, but a post-merge review would be appreciated!

File: cookiecutter/local_extensions.py [hunk 1]
```diff
@@ -123,6 +133,19 @@ def _get_random_weekday(self) -> str:
...
+
+        This will be faked when running the golden tests to make sure the tests are
+        reproduceable.
+
+        Returns:
+            The copyright year.
+        """
+        if self._is_golden_testing():
+            return 2023
+        return _datetime.datetime.now().year
```
Comment (daniel-zullo-frequenz) [inline cookiecutter/local_extensions.py:147]: Optionally we could have added `timezone` but it's not a big deal

**Review by daniel-zullo-frequenz — COMMENTED** [review]
I'm not sure if the world `fakable` exists. Anyway I had only an optional comment. LGTM

File: cookiecutter/local_extensions.py (hunk 1 above)
Comment (llucax) [reply cookiecutter/local_extensions.py:147]: No, we need the local time here, but it could have a comment to clarify.

**Pull request merged**